client/my-sites/backup/rewind-flow/granular-restore.tsx

```diff
@@ -0,0 +1,275 @@
+import { Button, Card, Gridicon } from '@automattic/components';
+import { Button as WordPressButton } from '@wordpress/components';
+import { Icon, arrowLeft, backup } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useCallback, useState } from 'react';
+import JetpackReviewPrompt from 'calypso/blocks/jetpack-review-prompt';
+import QueryJetpackCredentialsStatus from 'calypso/components/data/query-jetpack-credentials-status';
+import QueryRewindBackups from 'calypso/components/data/query-rewind-backups';
+import QueryRewindRestoreStatus from 'calypso/components/data/query-rewind-restore-status';
+import QueryRewindState from 'calypso/components/data/query-rewind-state';
+import { Interval, EVERY_FIVE_SECONDS } from 'calypso/lib/interval';
+import { useDispatch, useSelector } from 'calypso/state';
+import { areJetpackCredentialsInvalid } from 'calypso/state/jetpack/credentials/selectors';
+import { setValidFrom } from 'calypso/state/jetpack-review-prompt/actions';
+import { requestRewindBackups } from 'calypso/state/rewind/backups/actions';
+import { getInProgressBackupForSite } from 'calypso/state/rewind/selectors';
+import getBackupBrowserCheckList from 'calypso/state/rewind/selectors/get-backup-browser-check-list';
+import getInProgressRewindStatus from 'calypso/state/selectors/get-in-progress-rewind-status';
+import getRestoreProgress from 'calypso/state/selectors/get-restore-progress';
+import getRewindState from 'calypso/state/selectors/get-rewind-state';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { backupContentsPath } from '../paths';
+import Error from './error';
+import Loading from './loading';
+import ProgressBar from './progress-bar';
+import RewindFlowNotice, { RewindFlowNoticeLevel } from './rewind-flow-notice';
+import CheckYourEmail from './rewind-flow-notice/check-your-email';
+import type { RestoreProgress } from 'calypso/state/data-layer/wpcom/activity-log/rewind/restore-status/type';
+import type { RewindState } from 'calypso/state/data-layer/wpcom/sites/rewind/type';
+
+interface Props {
+	backupDisplayDate: string;
+	rewindId: string;
+	siteId: number;
+	siteUrl: string;
+}
+
+const BackupGranularRestoreFlow: FunctionComponent< Props > = ( {
```

Comment:
This is component is based on [client/my-sites/backup/rewind-flow/restore.tsx](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/backup/rewind-flow/restore.tsx), just removed everything that we don't require here (like the restore types selection).